app/views/shared/_header.html.haml

```diff
@@ -22,7 +22,7 @@
               %li.navbar-text= "#{acting_user.full_name} (#{acting_user.username})"
               %li= link_to "#{t('pages.cart')} (#{current_cart.order_details.count})", :cart
             - else
-              %li.navbar-text= "#{session_user.full_name} (#{session_user.username})"
+              %li.visible-with-link= link_to "#{session_user.full_name} (#{session_user.username})", user_user_preferences_path(session_user)
```

Comment:
This feels a little awkward to link to if you have no preferences (as there will be for NU/UIC for now). I wonder if it might be better to have it not link until you have some settings? I'm open to ideas.

Reply:
Which school has the password reset option available? I remember that being floated as something that might make sense to put on this page.

I like the idea of not linking unless there is something to do on the page, though.

Reply:
NU does if you're an external (non-netid) user, and dartmouth does for everyone since everyone is "external".

Reply:
On reflection, I'm going to call putting the reset password stuff onto the user preferences page out of scope for now.